docs/applications/readarr.mdx

```diff
@@ -0,0 +1,91 @@
+---
+id: readarr
+title: Readarr
+sidebar_label: Readarr
+---
+
+Readarr is a ebook (and maybe eventually magazine/audiobook) collection manager for Usenet and BitTorrent users. It can monitor multiple RSS feeds for new books from your favorite authors and will grab, sort and rename them.
+
+:::danger
+Readarr is in the early stages of development and is considered alpha. Install and use with caution.
+:::
+
+## Initial Setup
+
+Installing Readarr is easy. Simply issue the following command:
+
+```bash main
+sudo box install readarr
+```
+
+This command will configure readarr for your user.
+
+## How to Access
+
+After Readarr has been installed, it will be available for access at `https://<hostname.ltd>/readarr`.
```

Comment:
```suggestion
After Readarr has been installed, it will be available for access at `https://<hostname.ltd>/readarr`.

If you don't use `nginx`, you can find it on port `8787`.
```